web/admin.py

```
@@ -1,10 +1,13 @@
+from typing import Any, ClassVar, List, Optional, Type, Union, Dict
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)